sdk/formrecognizer/Azure.AI.FormRecognizer/src/DocumentModelAdministrationClient.cs

```diff
@@ -110,9 +110,10 @@ public DocumentModelAdministrationClient(Uri endpoint, TokenCredential credentia
             ServiceClient = new DocumentAnalysisRestClient(Diagnostics, pipeline, endpoint, options.VersionString);
         }
 
-        #region Build Document Model
```

Comment:
At first glance this file seems to have lots of changes but we're mostly reordering service methods. Changes are:
- Splitting service methods into different `#region` blocks. The regions we're using are aligned with the ones we use in the other client (`DocumentAnalysisClient`) and in tests: Document Models, Document Classifiers, and Management Operations.
- Making sure the async variant of a method is listed **immediately before** the sync variant. For example, `BuildDocumentAsync` should appear immediately before `BuildDocument`. This is the pattern we already use in the other client (`DocumentAnalysisClient`).
- Minor terminology changes in docstrings, such as "custom document analysis model" => "custom model".